CONTRIBUTIONS.md

```diff
@@ -0,0 +1,145 @@
+# Contributing 
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to all the repositories managed by Jordan Harband, which are hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+# Table of Contents 
```

Comment:
markdown files on github all have a "table of contents" button in the upper left corner now, so i don't think we need to repeat it here